IDE/Renesas/e2studio/RX72N/EnvisionKit/wolfssl_demo/key_data.c

```diff
@@ -0,0 +1,179 @@
+        0x3E, 0xF3, 0xD1, 0x7D, 0xBB, 0x33, 0x96, 0x40, 0x79, 0xC2, 0x7B, 0x15,
+        0x2E, 0xEE, 0xE3, 0x5B, 0x9C, 0x06, 0x72, 0x95, 0xFF, 0xCB, 0xC9, 0xE4,
+        0x96, 0x97, 0x18, 0x0D, 0xE7, 0x78, 0xCD, 0xE9, 0xA7, 0xEA, 0xE9, 0xDF
+    },
+    /* uint8_t encrypted_user_update_key[R_TSIP_AES256_KEY_BYTE_SIZE + 16]; */
+    {
+        0xD6, 0xDB, 0x33, 0x0B, 0xC4, 0x72, 0x5A, 0xD3, 0x23, 0xDE, 0x1C, 0xAF,
+        0xA8, 0x33, 0xA6, 0x85, 0xA0, 0xAB, 0xBF, 0x51, 0x8B, 0xF4, 0x68, 0x45,
+        0xBD, 0x6D, 0xA5, 0x7A, 0x88, 0x82, 0x9C, 0x50, 0x43, 0xD7, 0x53, 0xC0,
+        0x9E, 0xA8, 0x4B, 0x7E, 0x8A, 0x8B, 0xF3, 0x1B, 0xA5, 0x50, 0x55, 0x5D
+    },
+
+};
+
+/* Public key type of CA root cert: 0: RSA-2048 2: ECDSA-P256*/
+#if defined(USE_ECC_CERT)
+const uint32_t              encrypted_user_key_type =
+                                    R_TSIP_TLS_PUBLIC_KEY_TYPE_ECDSA_P256;
+#else
+const uint32_t              encrypted_user_key_type =
+                                    R_TSIP_TLS_PUBLIC_KEY_TYPE_RSA2048;
+#endif
+
+const unsigned char ca_ecc_cert_der_sig[] =
+{
+	0x53, 0x07, 0xba, 0xa4, 0xed, 0xd3, 0x8c, 0x2e, 0xea, 0x36, 0xe4, 0x3c, 
+    0x0c, 0x4a, 0x89, 0x6c, 0x39, 0x2c, 0xcf, 0xd6, 0x5d, 0x83, 0xe9, 0x3f,
+    0x12, 0x67, 0xdd, 0xbb, 0xa3, 0xe8, 0x33, 0x84,	0x70, 0x36, 0x24, 0xab,
+    0xb4, 0x7e, 0x79, 0x43, 0x77, 0xbf, 0x97, 0xb4, 0x34, 0xd3, 0xbb, 0x9b,
+	0xa5, 0x4c, 0x3c, 0xe0, 0xa6, 0xc2, 0x2a, 0xb0, 0x8b, 0xe1, 0x11, 0xf1,
+    0x7f, 0xf1, 0x7d, 0xad,	0x15, 0x10, 0x25, 0x89, 0x95, 0x6e, 0xdc, 0xcc,
+    0x8e, 0x1a, 0xf4, 0x6e, 0x48, 0xef, 0xe2, 0x17,	0xd5, 0x77, 0x4a, 0x07,
+    0x63, 0xf1, 0xe5, 0x63, 0x20, 0x7c, 0x20, 0x81, 0xda, 0x95, 0xca, 0x46,
+	0x86, 0x80, 0x6d, 0x68, 0x91, 0xc0, 0xd2, 0x85, 0xdf, 0x77, 0xf1, 0xb8,
+    0xa5, 0x6d, 0x3a, 0x09,	0x42, 0x60, 0xdb, 0xf0, 0x10, 0xd7, 0x74, 0xe9,
+    0x1c, 0xb7, 0xc5, 0x8d, 0x4e, 0x6a, 0x6c, 0x80,	0x75, 0x3b, 0xd5, 0xd1,
+    0xde, 0xbb, 0xd2, 0xfb, 0x35, 0x60, 0xa6, 0x31, 0x30, 0xc9, 0x79, 0x83,
+	0xfb, 0xce, 0xe1, 0x96, 0x7c, 0xb9, 0x12, 0x44, 0x1c, 0x96, 0x71, 0x02,
+    0x57, 0xd7, 0x97, 0xa0,	0xc0, 0x24, 0x8d, 0xf5, 0xc2, 0x5c, 0xbf, 0x59,
+    0x9a, 0x74, 0x85, 0xcb, 0x3f, 0x59, 0x58, 0x57,	0x8e, 0xf3, 0x9e, 0x25,
+    0x72, 0x61, 0xb7, 0xc1, 0x36, 0xa6, 0x76, 0xe4, 0x74, 0x2f, 0x1c, 0xbe,
+	0x0d, 0x33, 0x92, 0xba, 0xe7, 0xe7, 0x73, 0x35, 0x41, 0x96, 0x61, 0x95,
+    0xc0, 0x83, 0xab, 0xfd,	0x94, 0xe7, 0x4d, 0xea, 0xe9, 0x54, 0x7b, 0xe9,
+    0xfb, 0xcc, 0x4b, 0x8b, 0xbc, 0x7a, 0x47, 0x54,	0xa5, 0x64, 0x6b, 0x23,
+    0x7a, 0x06, 0x0f, 0x95, 0x3b, 0xd8, 0x5e, 0x32, 0x9d, 0xdf, 0x32, 0x74,
+	0x44, 0xc9, 0xa3, 0x7a, 0xe9, 0x38, 0xf6, 0x11, 0x6f, 0x59, 0x5d, 0xd1,
+    0x4e, 0xfe, 0xab, 0x55,
+};
+const int sizeof_ca_ecc_cert_sig = sizeof(ca_ecc_cert_der_sig);
+
+/* ./ca-cert.der.sign,  */
+const unsigned char ca_cert_der_sig[] =
+{
+	0x8b, 0x61, 0x86, 0xe5, 0x0e, 0x2b, 0x4c, 0x76, 0x42, 0x4b, 0x3d, 0x75,

```

Comment:
Please remove hard tabs in ca_cert_der_sig[].

Reply:
Replaced tabs with white spaces.